README.md

```diff
@@ -37,36 +37,27 @@ First, create a container for `traefik-pages`:
       - SITES_ROOT=/mnt/sites
       - TRAEFIK_SERVICE=traefik-pages@docker
       - AUTH_PASSWORD=hunter2
+    labels:
+      - traefik.enable=true
+      - traefik.http.services.traefik-pages.loadbalancer.server.port=5000
 ```
 
 This doesn't need to be in the same file as Traefik, but it does need to be accessible to Traefik using a fixed hostname and IP. If Traefik is running in host mode (as I do), you'll need to bind `traefik-pages` to an internal interface, and listen to that.
 
-Next, you'll need to create a HTTP and file provider for Traefik, using the ports and password previously configured.
+The labels enable traefik autoconfiguration and configure the service listener. Note that the service name should match `$TRAEFIK_SERVICE`. The second label can be omitted, but $TRAEFIK_SERVICE will need to match the automatically configured name of the service.
```

Comment:
```suggestion
The label enables traefik autoconfiguration to detect `traefik-pages`. Note that `$TRAEFIK_SERVICE` must match the service name created by traefik. 
```

Reply:
Thanks, I couldn't figure out how to word that in a way that made sense.